src/main/kotlin/no/nav/syfo/personhendelse/PersonhendelseService.kt

```diff
@@ -0,0 +1,53 @@
+package no.nav.syfo.personhendelse
+
+import no.nav.person.pdl.leesah.Endringstype
+import no.nav.person.pdl.leesah.Personhendelse
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.domain.PersonIdentNumber
+import no.nav.syfo.oppfolgingstilfelle.OppfolgingstilfelleService
+import no.nav.syfo.personhendelse.db.createPerson
+import no.nav.syfo.personhendelse.db.getDodsdato
+import no.nav.syfo.util.kafkaCallId
+import org.slf4j.LoggerFactory
+import java.util.UUID
+
+class PersonhendelseService(
+    private val database: DatabaseInterface,
+    private val oppfolgingstilfelleService: OppfolgingstilfelleService,
+) {
+    suspend fun handlePersonhendelse(personhendelse: Personhendelse) {
+        if (personhendelse.doedsfall != null && personhendelse.endringstype == Endringstype.OPPRETTET) {
```

Comment:
Det som er vanskelig med `Endringstype` er at det kan komme annulleringer på en endring. Hva om noen "dør" og så viste det seg å være feil, den var ikke død likevel 🥴 

Sånn vi løste det i `syfooversiktsrv` var vel å bare slette `navn` uansett hvilken endring som kom inn, og så heller populere den dataen på neste oppslag mot personen. Det vil vel ikke funke her fordi man ikke gjør oppslag 🤔 Skulle man da tillat alle endringstyper? Jeg vet ikke hva dataen inneholder dersom det kommer en annullering på dødsdato 🤔

Reply:
Ja, det forekommer en veldig sjelden gang at et dødsfall annulleres, men jeg vet ikke om det er noe vi trenger å ta høyde for her?